build.sbt

```diff
@@ -50,10 +50,29 @@ lazy val aggregatedProjects: Seq[ProjectReference] = Seq(
   discovery
 )
 
+/** TODO fix remoteTests in job conf and add remote coverage back.
+  * Does not include
+  * - planned removals in 2.6 https://github.com/akka/akka/milestone/119
+  * - docs?, protobuf, benchJmh
+  */
+lazy val aggregatedCoverage: Seq[ProjectReference] = Seq(
+  actor, actorTests,
+  cluster, clusterMetrics, clusterSharding, clusterTools, contrib,
+  discovery, distributedData, multiNodeTestkit, osgi,
+  persistence, persistenceQuery, persistenceShared, persistenceTck,
+  slf4j, stream, streamTestkit, streamTests, streamTestsTck, testkit,
+  actorTyped, actorTypedTests, actorTestkitTyped,
+  persistenceTyped, clusterTyped, clusterShardingTyped,
+  streamTyped)
```

Comment:
The thinking here is it is cleaner to be declarative up front on what we instrument for coverage in the nightly reports, versus adding `coverageEnabled := false` in the modules below which we exclude. My preference is consolidate in one location vs sprinkle coverage related things in `build.sbt`.

Reply:
I didn't think of that option. To be honest I kind of prefer that, since that is more like how all the other crosscutting aspects are managed. (Publish, Mima, Whitesource etc)

Reply:
Ok, let me push that.